src/parser.rs

```diff
@@ -4926,4 +4910,124 @@ mod tests {
             assert_eq!(ast.to_string(), sql.to_string());
         });
     }
+
+    #[test]
+    fn custom_prefix_parser() -> Result<(), ParserError> {
```

Comment:
I recommend putting these tests into `tests` somewhere so that we can be sure everything needed is exposed `pub`lically.

Perhaps `tests/sqlparser_custom_dialects.rs`?

Reply:
Good idea. I have moved them.